### PR TITLE
Upgrade to powsybl network-store 1.3.0

### PR DESCRIPTION
**Pull request by ayolab**

### CODE REVIEW

File: src/test/java/org/gridsuite/modification/server/Impacts/TestImpactUtils.java [hunk 1]
```diff
@@ -145,10 +145,14 @@ public static void testBranchImpacts(ObjectMapper mapper, SimpleImpactType impac
                                          String breakerId2, String disconnectorId2, String substationId2) throws JsonProcessingException {
         Optional<NetworkModificationResult> networkModificationResult = mapper.readValue(resultAsString, new TypeReference<>() { });
         assertTrue(networkModificationResult.isPresent());
+        List<SimpleElementImpact> impacts = createBranchImpacts(impactType, branchType, branchId, breakerId1, disconnectorId1, substationId1, breakerId2, disconnectorId2, substationId2);
+        if (impactType != SimpleImpactType.DELETION) {
```
Comment (SlimaneAmar) [inline src/test/java/org/gridsuite/modification/server/Impacts/TestImpactUtils.java:149]: impactType == SimpleImpactType.CREATION ?

File: src/test/java/org/gridsuite/modification/server/Impacts/TestImpactUtils.java [hunk 2]
```diff
@@ -145,10 +145,14 @@ public static void testBranchImpacts(ObjectMapper mapper, SimpleImpactType impac
...
+            impacts.add(createElementImpact(SimpleImpactType.MODIFICATION, branchType, branchId, new TreeSet<>(List.of(substationId1, substationId2)))); // case with newtapChanger
```
Comment (SlimaneAmar) [inline src/test/java/org/gridsuite/modification/server/Impacts/TestImpactUtils.java:150]: Move this in `createBranchImpacts` method ?

File: src/test/java/org/gridsuite/modification/server/Impacts/TestImpactUtils.java (hunk 1 above)
Comment (ayolab) [reply src/test/java/org/gridsuite/modification/server/Impacts/TestImpactUtils.java]: Done

File: src/test/java/org/gridsuite/modification/server/Impacts/TestImpactUtils.java (hunk 2 above)
Comment (ayolab) [reply src/test/java/org/gridsuite/modification/server/Impacts/TestImpactUtils.java]: Done

**Review by SlimaneAmar — APPROVED** [review]

**Review by dbraquart — APPROVED** [review]
code: ok

**Pull request merged**